packages/amplication-client/src/Components/CreateResourceButtonItem.tsx

```diff
@@ -16,10 +18,26 @@ const CreateResourceButtonItem = ({ item }: props) => {
 
   const { currentWorkspace, currentProject } = useContext(AppContext);
 
+  const handleCreateResourceByType = {
+    Service: () => {
+      trackEvent({
+        eventName: AnalyticsEventNames.CreateService,
+      });
+    },
+    MessageBroker: () => {
+      trackEvent({
+        eventName: AnalyticsEventNames.CreateMessageBroker,
+      });
+    },
+    ProjectConfiguration: () => {
+      trackEvent({
+        eventName: AnalyticsEventNames.CreateProjectConfiguration,
+      });
+    },
+  };
+
```

Comment:
I think it makes more sense to use the exact same mapping but only with the eventname instead of including the trackEvent function 

```suggestion
  const RESOURCE_TYPE_TO_EVENT_NAME = {
    Service: AnalyticsEventNames.CreateService,
    MessageBroker: AnalyticsEventNames.CreateMessageBroker,
    ProjectConfiguration: AnalyticsEventNames.CreateProjectConfiguration,
  };

```